src/components/SponsorsSection/Sponsor.jsx

```diff
@@ -66,7 +66,7 @@ function Sponsor() {
       id='sponsors'
       title={sponsors.title}
       description={sponsors.content}
-      link={{ text: 'Sponsor Us', link: links.sponsor }}
+      link={{ text: 'Become a community partner.', link: links.sponsor }}
     >
```

Comment:
Revert this as well. The changes weren't to be done here. We have a dedicated sponsor us section now, check the live website ones (just below the hero section).